Replicator/ReplicatorOptions.hh

```diff
@@ -158,6 +175,120 @@ namespace litecore { namespace repl {
         bool boolProperty(slice property) const   {return properties[property].asBool();}
 
         explicit operator std::string() const;
+
+        // Collection Options:
+
+        static alloc_slice collectionSpecToPath(C4CollectionSpec spec, bool omitDefaultScope=true) {
```

Comment:
Perhaps add a comment about the reasoning behind this involving the BLIP protocol.